bin/check-memory-percent.sh

```diff
@@ -48,15 +48,26 @@ fi
 WARN=${WARN:=80}
 CRIT=${CRIT:=90}
 
-#Get total memory available on machine
-TotalMem=$(free -m | grep Mem | awk '{ print $2 }')
-#Determine amount of free memory on the machine
-set -o pipefail
-FreeMem=$(free -m | grep buffers/cache | awk '{ print $4 }')
-if [ $? -ne 0 ];
+os=$(uname)
```

Comment:
Could you tell me what do you mean with this request? I don't understand it

Reply:
below you had:
```
if [ os.chomp = "Darwin" ];
then
  # foo
elif [ os.chomp ];
then
  # bar
```

If you are gonna do something like that it would make more sense to do the chmop when you store the variable (though bash does not have chomp in the first place) rather than repeating it each time you do a comparison.

Reply:
yes, you are right. I have just delete chomp because it doesn't work in bash. I think it is fine now.